.changeset/brown-lizards-remain.md

```diff
@@ -0,0 +1,5 @@
+---
+'@commercetools-uikit/card': patch
```

Comment:
Maybe we need to recreate the changeset since now we're changing a different component.

Reply:
Ah true, updated here: https://github.com/commercetools/ui-kit/pull/2624/commits/ba411e4fc09d409573abad10eed42803dc69e915

Thank you